problems/variables.js

```diff
@@ -44,6 +84,16 @@
 //  * Find the average grade of all students
 //  * Print out if Dee's grade is higher than the class average
 
+// let alicesGrade = 90
+// let bobsGrade = 80
+// let camsGrade = 85
+// let averageGrade = (alicesGrade + bobsGrade + camsGrade) / 3
+// console.log ( "The average grade is " + averageGrade )
+// let deesGrade = 95
+//  averageGrade = (alicesGrade + bobsGrade + camsGrade + deesGrade) / 4
+// console.log ( " The average grade of all the  students is " + averageGrade )
+// console.log ( " Dees grade of " + deesGrade + " is higher than the class average of " + averageGrade )
```

Comment:
I'm going to challenge you to use a conditional statement here. Google condition statements javascript

Reply:
Challenge Accepted!! 

let alicesGrade = 90
let bobsGrade = 80
let camsGrade = 85
let averageGrade = (alicesGrade + bobsGrade + camsGrade) / 3
console.log ( "The average grade is " + averageGrade )
let deesGrade = 95

averageGrade = (alicesGrade + bobsGrade + camsGrade + deesGrade) / 4
    
console.log ( " The average grade of all the  students is " + averageGrade )

    if (deesGrade > averageGrade){
    console.log("Dee's grade is higher than the class average");
}       else {
    console.log("Dee's grade is not higher than the class average");
}